src/Adapter/MSTest.CoreAdapter/RunConfigurationSettings.cs

```diff
@@ -29,7 +29,7 @@ public class RunConfigurationSettings
         /// </summary>
         public RunConfigurationSettings()
         {
```

Comment:
Was talking to @harshjain2 yesterday. Should this settings also be exposed through MSTestSettings something like `MSTestSettings.RunConfiguration.CollectSourceInformation`. We will then have a one stop please instead of searching in multiple places.